src/test/javascript/spec/component/shared/metis/postings-header/post-header/post-header.component.spec.ts

```diff
@@ -96,4 +97,21 @@ describe('PostHeaderComponent', () => {
         expect(getElement(debugElement, '.editIcon')).not.toBe(null);
         expect(getElement(debugElement, '.deleteIcon')).not.toBe(null);
     });
+
+    it.each`
+        input                  | expect
+        ${UserRole.INSTRUCTOR} | ${'fa fa-user-graduate'}
+        ${UserRole.TUTOR}      | ${'fa fa-user-check'}
+        ${UserRole.USER}       | ${'fa fa-user'}
+    `('should display relevant icon and tooltip for author authority', (param: { input: UserRole; expect: string }) => {
+        component.posting = metisAnnouncement;
+        component.posting.authorRole = param.input;
+        component.ngOnInit();
+        fixture.detectChanges();
+
+        // should display relevant icon for author authority
+        const icon = getElement(debugElement, 'fa-icon');
+        expect(icon).not.toBe(null);
```

Comment:
```suggestion
        expect(icon).not.toBeNull();
```
sorry :)

Reply:
No worries, thanks for pointing it out :)